db/migrate/20231103061213_add_terms_of_service_accepted_at_to_spree_users.rb

```diff
@@ -0,0 +1,13 @@
+class AddTermsOfServiceAcceptedAtToSpreeUsers < ActiveRecord::Migration[7.0]
+  def up
+    add_column :spree_users, :terms_of_service_accepted_at, :datetime
+
+    if Spree::Config.enterprises_require_tos == true
+      Spree::User.update_all(terms_of_service_accepted_at: Time.zone.now)
```

Comment:
This is for all users including shoppers. But I think that only enterprise users accept ToS.